package.json

```diff
@@ -35,7 +35,7 @@
     "file-loader": "^6.2.0",
     "html-webpack-plugin": "^4.5.0",
     "jest": "^26.6.1",
-    "preload-webpack-plugin": "^3.0.0-beta.4",
+    "preload-webpack-plugin": "^3.0.0-beta.3",
     "sass": "^1.27.0",
     "sass-loader": "^10.0.4",
     "style-loader": "^2.0.0",
```

Comment:
- [ ] There are a lot of unnecessary dependencies here, please remove them and leave **ONLY** jest.